mssql-plugin/src/main/java/io/cdap/plugin/mssql/SqlServerConnector.java

```diff
@@ -152,7 +153,16 @@ protected String getStratifiedQuery(String tableName, int limit, String strata,
   @Override
   protected Schema getSchema(int sqlType, String typeName, int scale, int precision, String columnName,
                              boolean isSigned, boolean handleAsDecimal) throws SQLException {
-    if (SqlServerSourceSchemaReader.shouldConvertToDatetime(typeName)) {
+    if (Types.TIMESTAMP == sqlType) {
+      // SmallDateTime case
```

Comment:
This should ideally cover Datetime, Datetime2 and SmallDatetime all 3 of them.

Reply:
done